.github/workflows/test-package.yml

```
@@ -34,3 +34,39 @@ jobs:
       # go1.12 vet shows spurious 'unknown identifier' issues
       if: matrix.go-version != '1.7' && matrix.go-version != '1.8' && matrix.go-version != '1.9' && matrix.go-version != '1.10' && matrix.go-version != '1.12'
       run: go vet . ./gin ./martini ./negroni ./sessions ./headers ./errors
+
+    - name: install integration dependencies
+      run: |
+        sudo apt-get install docker-compose
+        sudo gem install bundler
+        bundle install
+    - name: maze tests - plain Go apps
+      env:
+        GO_VERSION: ${{ matrix.go-version }}
+      run: bundle exec bugsnag-maze-runner --color --format progress features/plain_features features/net_http_features
+    - name: maze tests - Martini apps
+      env:
+        GO_VERSION: ${{ matrix.go-version }}
+      run: bundle exec bugsnag-maze-runner --color --format progress features/martini_features
+    - name: maze tests - Gin apps
+      # gin requires go1.11+
+      if: matrix.go-version != '1.7' && matrix.go-version != '1.8' && matrix.go-version != '1.9' && matrix.go-version != '1.10'
```

Looking at [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#operators), I _think_ this can be simplified (also elsewhere):

```suggestion
      if: matrix.go-version < 1.11
```

---

I tried though `1.11` is considered less than `1.7` through number coercion.

---

☹️ 

that does makes sense though

---

I was also a little sad.